src/components/AddProfileInformation.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  FunctionComponent,
+  KeyboardEvent,
+  useCallback,
+  useEffect,
+  useRef,
+} from "react";
+import { Box } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { StringMap } from "common";
+import NEWMLogo from "assets/images/NEWMLogo";
+import { FilledButton, Typography } from "elements";
+import { useFormikContext } from "formik";
+import { FilteredTagsField, GradientTextInputField } from "components";
+
+interface AddProfileInformationProps {
+  readonly fieldName: string;
+  readonly nextRoute: string;
+  readonly prompt: string;
+  readonly helperText?: string;
+  readonly tags?: ReadonlyArray<string>;
+}
+
+const AddProfileInformation: FunctionComponent<AddProfileInformationProps> = ({
+  fieldName,
+  nextRoute,
+  prompt,
+  tags,
+  helperText = "",
+}) => {
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  const navigate = useNavigate();
+
+  const { errors, handleBlur } = useFormikContext<StringMap>();
+
+  const isValid = !errors[fieldName];
```

Comment:
There is an `isValid` boolean returned from `useFormikContext`, but it's for the whole field. Formik is primarilly designed for single page forms, but because we're creating a multi-step form, it adds some complexity.

Reply:
ohh i see, i missed the error destructure